tests/duplicate-check-inputs/default.nix

```diff
@@ -3,7 +3,7 @@
 }:
 
 buildPythonPackage {
-  pname = "patch-phase";
+  name = "patch-phase";  
```

Comment:
Would editorconfig help in avoiding the trailing spaces?

Reply:
Hehe, sorry I keep making them. Sure.